pkg/collect/helm.go

```diff
@@ -57,9 +57,11 @@ func (c *CollectHelm) Collect(progressChan chan<- interface{}) (CollectorResult,
 
 	output := NewResult()
 
-	releaseInfos, err := helmReleaseHistoryCollector(c.Collector.ReleaseName, c.Collector.Namespace)
+	releaseInfos, err := helmReleaseHistoryCollector(c.Collector.ReleaseName, c.Collector.Namespace, c.Collector.CollectValues)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to get Helm release history")
+		output.SaveResult(c.BundlePath, "helm/errors.json", marshalErrors(err))
+		klog.Error(err)
+		return nil, nil
```

Comment:
The `error` type does not get marshalled correctly

```suggestion
		errsToMarshal := []string{}
		for _, e := range err {
			errsToMarshal = append(errsToMarshal, e.Error())
		}
		output.SaveResult(c.BundlePath, "helm/errors.json", marshalErrors(errsToMarshal))
		klog.Errorf("error collecting helm release info: %v", err)
		return output, nil
```